examples/snippets-filter/README.md

```diff
@@ -1,3 +1,3 @@
 # SnippetsFilter
 
-This directory contains the YAML files used in the [Use the SnippetsFilter API](https://docs.nginx.com/nginx-gateway-fabric/how-to/traffic-management/snippets-filters/) guide.
+This directory contains the YAML files used in the [Use the SnippetsFilter API](https://docs.nginx.com/nginx-gateway-fabric/how-to/traffic-management/snippets/) guide.
```

Comment:
I would change this link to say `SnippetsFilter API` for a better flow. I don't think the link always has to match the exact name of the doc.

Reply:
The link doesn't have to match the exact name of the document, and shorter URLs are usually preferable - appending filters makes it highly explicit, but unnecessary.